natlib/svcd.c

```diff
@@ -185,3 +187,84 @@ static int svcd_init( lua_State *L )
 
     return 0;
 }
+
+// Lua: storm.n.svcd_add_service ( svcd_id )
+// Add a new service to the service daemon
+// this must be called before SVCD.advertise()
```

Comment:
I know this comment was pulled from the lua version, but the current implementation advertises automatically so this is no longer needed (in either lua or c).